lib/matplotlib/backends/backend_wx.py

```diff
@@ -1826,9 +1826,49 @@ def remove_rubberband(self, dc=None):
             self._rect = None
 
 
+class _TableDialog(wx.Dialog):
+    def __init__(self, parent, help, title="Help"):
```

Comment:
`help` is built-in function. While technically you can use that variable name, I would not advise to do so. Also, it's slightly imprecise. Maybe rename to `help_entries`.